src/parser/jobs/whm/Gauge.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable */
```

Comment:
Nuh-uh.
Probably left over from dev or w/e, code looks fine at a glance.
Doubt it'll flag much but I am gonna req that be removed.

Reply:
I gotta be careful with those, thanks for spotting it